MySQL/deployment.yaml

```diff
@@ -0,0 +1,89 @@
+---
+# Source: bluecompute-ce/charts/mysql/templates/deployment.yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: inventory-mysql
+  labels:
+    app: inventory-mysql
+    chart: "mysql-0.15.0"
+    release: "bluecompute"
+    heritage: "Helm"
+spec:
+  template:
+    metadata:
+      labels:
+        app: inventory-mysql
+    spec:
+      initContainers:
+      - name: "remove-lost-found"
+        image: "busybox:latest"
+        imagePullPolicy: "IfNotPresent"
+        command:  ["rm", "-fr", "/var/lib/mysql/lost+found"]
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/mysql
+      # - name: do-something
```

Comment:
Commented code with no description on why it's commented should be avoided, if possible.

Reply:
Used it for testing ... removed it now